tests/wpt/web-platform-tests/html/semantics/embedded-content/media-elements/event_pause_noautoplay.html

```diff
@@ -21,7 +21,13 @@
     t.done();
   }, false);
   a.src = getAudioURI("/media/sound_5") + "?" + new Date() + Math.random();
-  a.play();
+  a.play()
```

Comment:
This should use one of:

- t.step_func and make this test an async_test
- promise_rejects(t, ...) and make this test a promise_test
- just .catch(() => {}) it because testing the rejection is not the point of the test.

Same for all other modified tests.

Reply:
@domenic thanks for review! The second solution for using `promise_rejects` looks better for me because the `pause()` will make `play` rejected; I think it should be fine to test it with rejected promise. Will update those tests, thanks :)

Reply:
@domenic The tests are updated!

I only used `promise_rejects` for `event_pause_noautoplay.html` and `event_play_noautoplay.html`; for other tests, I just used the 3rd solution to `.catch(() => {})`.

Please help to review them again! Thanks! 🙇